src/app/find-help/view-offer-transport/view-offer-transport.component.ts

```diff
@@ -15,12 +15,15 @@ export class ViewOfferTransportComponent implements OnInit {
   offerId: number = 0;
   data = defaults<TransportOffer>();
   categoryRouteName = CategoryRoutingName.TRANSPORT;
+  redirectedFromAccount: boolean;
   constructor(
     private router: Router,
     private route: ActivatedRoute,
     private transportResourceService: TransportResourceService,
     private storeUrlService: StoreUrlService
-  ) {}
+  ) {
+    this.redirectedFromAccount = this.router.getCurrentNavigation()?.extras?.state!['redirectFromAccount'];
```

Comment:
Let's use dot notation consistently:
```suggestion
    this.redirectedFromAccount = this.router.getCurrentNavigation()?.extras?.state!.redirectFromAccount;
```

Reply:
Property 'redirectFromAccount' comes from an index signature, so it must be accessed with ['redirectFromAccount'].